src/game/boe.newgraph.cpp

```diff
@@ -124,7 +124,7 @@ void apply_unseen_mask() {
 	rectangle base_rect = {9,9,53,45},to_rect,big_to = {13,13,337,265};
 	bool need_bother = false;
 	
-	if(!get_bool_pref("DrawTerrainFrills", true) || fog_lifted)
+	if(fog_lifted)
```

Comment:
This one is… odd. By a strict reading, deleting the preference should follow this path:

* "if not DrawTerrainFrills or fog_lifted then return"
* "if not false or fog_lifted then return"
* "if true or fog_lifted then return"
* "return"

In other words, deleting _the entire function_. But that doesn't sound correct at all, so I suspect this check was wrong to begin with. I wonder if theory is supported by the original source code…?